src/foam/nanos/auth/login/LoginView.js

```diff
@@ -0,0 +1,477 @@
+/**
+ * @license
+ * Copyright 2020 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.nanos.auth.login',
+  name: 'LoginView',
+  extends: 'foam.u2.View',
+
+  imports: [
+    'appConfig',
+    'clientLoginService',
+    'loginVariables',
+    'memento',
```

Comment:
I'm getting `debug.js:425 Assertion failed: Missing required import: memento in foam.nanos.auth.login.LoginView`

I don't think we have memento available at logged out time.  Also looks like we dont' actually use it so we should probably remove the import?

Reply:
removed.